tests/acceptance/test_mender-connect.py

```diff
@@ -305,11 +330,11 @@ def test_mender_connect_reconnect(
             )
 
             dbus_set_token_and_url(connection, "", "")
-            kill_time = qemu_system_time(connection)
             # kill the server and wait for error
             with_mock_servers[1].kill()
+            kill_time = qemu_system_time(connection)
```

Comment:
This change is probably unnecessary and I vote for reverting it. It is in effect the same thing, but capturing the time _before_ the action (in this case before killing the server) makes it consistent across the test.

Reply:
Fixed it now :)